changes/ce/feat-12719.en.md

```diff
@@ -0,0 +1,11 @@
+Support multiple clientid and username Query string parameters in "/clients" API.
+Add an option to specify which client info fields must be included in the response.
```

Comment:
this is a great feature which deserves its own PR and release note ;)

Reply:
Split into two blocks as discussed.